lib/scrivener/paginater/ecto/query.ex

```diff
@@ -10,12 +10,11 @@ defimpl Scrivener.Paginater, for: Ecto.Query do
         page_size: page_size,
         page_number: page_number,
         module: repo,
-        caller: caller,
```

Comment:
I can not find `:caller` as an option of the repository in the docs

Reply:
I believe that is scomething from Scrivener core, not Ecto

https://github.com/mojotech/scrivener/blob/09ab5c2b2b474bd1dca71723274b674a1f1805f3/lib/scrivener/config.ex#L3

https://github.com/mojotech/scrivener/commit/e7a68e2dae7109796e0596b2e54a93f29038c186

Why it was originally added to Scrivener and if it was related to some underlying option in Ecto, I couldn't say.  It doesn't appear to be a shared module option of `Ecto.Repo`  anymore however

Reply:
There does appear to be some underlying references to it inside Ecto as well

https://github.com/search?q=repo%3Aelixir-ecto%2Fecto+caller&type=code

Reply:
Should I merge it into the `options` anyway? Just in case, but overall, I think the refactoring is worth it;

Reply:
Yes, I'd keep it.  If we are uncertain its original intended usage, I'd prefer not to drop it on the floor.  We can keep backwards compatibility until the implications of dropping it completely are known.

Reply:
Like that?

Reply:
I guess let me take a step back.  What issues where/are you having versus the original PR #102 and how it kept/uses the caller argument?  Is there a compilation or invocation failure when doing so that necessitated its removal?

I've yet to run this code and/or associated tests, so I've only been looking at this per this PR and #102.

Reply:
What removal? I put the caller back